pkg/lightning/restore/restore.go

```diff
@@ -105,6 +105,7 @@ const (
 		task_id BIGINT(20) UNSIGNED NOT NULL,
 		pd_cfgs VARCHAR(2048) NOT NULL DEFAULT '',
 		status  VARCHAR(32) NOT NULL,
+		state   TINYINT(1) NOT NULL DEFAULT 0 COMMENT '0: normal, 1: exited before finish',
```

Comment:
should the `taskMetaTableName` be changed?

otherwise if we used Lightning before, and this CREATE TABLE IF NOT EXISTS means the `task_meta` with the *old* table structure will be used

Reply:
Not sure. But since we don't GA this feature and if lightning exited before finished, the current logic may still not be recover except manually drop the meta schema. We also don't recommend change lightning binary during one import task.